ask.py

```diff
@@ -954,6 +954,15 @@ def set_boilerplate():
 	flask_boilerplate += '\t\t\t\ttarget[key] = source[key]\n'
 	flask_boilerplate += '\n\t\treturn target\n'
 
+	flask_boilerplate += '\n\t@staticmethod\n'
```

Comment:
Move the whole function under the quickSet() function just so that quick_set() and quickSet() after after each other.